client/src/elm/Translate.elm

```diff
@@ -19065,6 +19145,96 @@ translationSet trans =
+            , kirundi = Nothing
+            }
+
+        ResilienceMessageGrowth5Paragraph2 ->
+            { english = "Remember: Every day may not be good, but there will be something good in every day."
+            , kinyarwanda = Just "Imihigo irakomeye kandi irakomeje/irashoboka. Kandi Buhoro buhoro nirwo rugendo."
+            , kirundi = Nothing
+            }
+
+        ResilienceMessageGrowth6Title ->
+            { english = "Make a list of things you value and want to achieve"
+            , kinyarwanda = Just "Kora urutonde rw'ibyo uha agaciro kandi ushaka kugeraho"
+            , kirundi = Nothing
+            }
+
+        ResilienceMessageGrowth6Paragraph1 ->
+            { english = "Think about what you value most in life. Is it being a good parent? Is it helping your community? Is it your career? Plan activities that help you do what is most important to you. If you want to be a good parent, make sure you spend an hour playing with your child every day."
+            , kinyarwanda = Just "Tekereza kubyo uha agaciro cyane mu buzima. Ese ni ukuba umubyeyi mwiza? Ese ni ugufasha abaturage? Ese ni umwuga wawe? Tegura ibikorwa bigufasha gukora ibifite akamaro. Niba ushaka kuba umubyeyi mwiza, mara nibura isaha ukina n'umwana wawe buri munsi."
+            , kirundi = Nothing
+            }
+
+        ResilienceMessageGrowth6Paragraph2 ->
+            { english = "Remember: Live a life of purpose."
+            , kinyarwanda = Just "Baho ubuzima bufite intego."
+            , kirundi = Nothing
+            }
+
+        ResilienceMessageGrowth7Title ->
+            { english = "Here is a time of positivity"
+            , kinyarwanda = Just "Iki ni igihe cyo gutekereza ibiguteza imbere"
+            , kirundi = Nothing
+            }
+
+        ResilienceMessageGrowth7Paragraph1 ->
+            { english = "When something unexpected happens, you may have thoughts like ‘I’m bad’ or ‘It’s all my fault.’ Thinking negative thoughts about yourself can affect your mood, your work, and your relationships. You can practice thinking more positive thoughts. Instead of ‘It’s my fault’, try thinking ‘I made a mistake, but I have learned a lesson.’ You are in charge of your thoughts."
+            , kinyarwanda = Just "Iyo ikintu gitunguranye kibaye, ushobora kugira ibitekerezo uti: \"Ndi mubi\", ni \"amakosa yanjye\". Kugira ibitekerezo bitari byiza kuri wowe bishobora kugira ingaruka ku myumvire yawe, ku kazi kawe, n'umubano wawe n'abandi. Ushobora kwitoza kugira ibitekerezo byiza. Aho kugira ngo uvuge \"Ni amakosa yanjye\", gerageza utekereze uti \"Nakoze ikosa, ariko nakuyemo isomo\". Ni wowe uyobora ibitekerezo byawe."
+            , kirundi = Nothing
+            }
+
+        ResilienceMessageGrowth7Paragraph2 ->
+            { english = "Remember: Praise loudly, blame softly."
+            , kinyarwanda = Just ""
```

Comment:
@anvmn here it is because we have a text in english but in Kinyarwanda they don't have a text. And asked us where there is not saying in kinyarwanda we skip the text